tests/simulator/initialize/test_init_mhd.py

```
@@ -0,0 +1,150 @@
+
+from tests.diagnostic import all_timestamps
+from tests.simulator.test_initialization import InitializationTest
+
+
+class MHDInitializationTest(InitializationTest):
+    def getHierarchy(
+        self,
+        ndim,
+        interp_order,  # torm?
+        qty,
+        refinement_boxes={},
```

Ruff (0.15.10)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)